objects/src/accounts/account_id.rs

```diff
@@ -308,6 +313,23 @@ impl From<AccountId> for LeafIndex<ACCOUNT_TREE_DEPTH> {
 // CONVERSIONS TO ACCOUNT ID
 // ================================================================================================
 
+pub const fn account_id_from_felt(value: Felt) -> Result<AccountId, AccountError> {
```

Comment:
I added a few `const` functions. My goal was to remove the duplication in the test context, when setting up account ids and assets. The issue is that `unwrap` is no `const` yet, so I ended up leaving the code duplication behind.

I left the changes that introduce the const function, which can be used once `unwrap` constness is stabilized.